src/content/blog/2018-01-29-simplifying-interledger-the-graveyard-of-possible-protocol-features.md

```diff
@@ -1,11 +1,12 @@
 ---
-layout: ../../layouts/BlogLayout.astro
 title: "Simplifying Interledger: The Graveyard of Possible Protocol Features"
 description: As the development of the Interledger Protocol (ILP) nears completion, I thought we should take a moment to remember some of the many core protocol features we’ve killed off along the way.
 date: 2018-01-29
 slug: simplifying-interledger-the-graveyard-of-possible-protocol-features
-authors: [Evan Schwartz]
-author_urls: [https://www.linkedin.com/in/evanmarkschwartz/]
+authors:
```

Comment:
Realised that we were doing tags in one format, and authors in another format, so I just standardised all YAML sequences (AKA arrays) to the same format.